infrastructure/simulation-tool/src/operations/withdraw.ts

```diff
@@ -0,0 +1,112 @@
+import { Wallet as RollupWallet, Transaction, utils } from '@rsksmart/rif-rollup-js-sdk';
+import { sleep } from '@rsksmart/rif-rollup-js-sdk/build/utils';
+import { BigNumber } from 'ethers';
+import { getRandomBigNumber } from '../utils/number.utils';
+import config from '../utils/config.utils';
+import { getRandomElement } from './common';
+import { TransactionReceipt } from '@rsksmart/rif-rollup-js-sdk/build/types';
+
+type PreparedWithdrawal = Parameters<RollupWallet['withdrawFromSyncToRootstock']>[number] & {
+    amount: BigNumber;
```

Comment:
can remove this.

Reply:
I kept this to avoid dealing with`BigNumberish`